src/modules/numberplate/types.skip.ts

```diff
@@ -69,6 +69,7 @@ export interface PlateResultApi {
  */
 export type PlateOptions = string | PlateApi;
 export type ProvinceObject = Array<{ province: string; codes: Array<number> }>;
+export type multiLangProvince = { fa: string | Array<string>, en: string | Array<string> };
```

Comment:
You've written this type, but you didn't use it in the `findPlateProvince` return type.